linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_unlock_time/rule.yml

```diff
@@ -58,6 +59,6 @@ ocil_clause: 'unlock_time is less than the expected value'
 ocil: |-
     To ensure the failed password attempt policy is configured correctly, run the following command:
     <pre>$ grep pam_faillock /etc/pam.d/system-auth</pre>
-    The output should show <tt>unlock_time=&lt;some-large-number&gt;</tt> or <tt>never</tt>.
+    The output should show <tt>unlock_time=&lt;some-large-number&gt;</tt> or <tt>0</tt> for <tt>never</tt>.
```

Comment:
Can ``0`` be used across all RHEL versions, or should RHEL 6 still have ``never``?

Reply:
@shawndwells looks like this was also finally fixed in the latest version of RHEL6

Reply:
Thanks for double checking.